components/UI/actions/socialmediaActions.tsx

```diff
@@ -11,45 +12,42 @@ type SocialMediaActionsProps = {
 const SocialMediaActions: FunctionComponent<SocialMediaActionsProps> = ({
   identity,
 }) => {
-  const [apiIdentity, setApiIdentity] = useState<Identity | undefined>();
+  const [twitter, setTwitter] = useState<string | undefined>();
+  const [discord, setDiscord] = useState<string | undefined>();
+  const [github, setGithub] = useState<string | undefined>();
 
   useEffect(() => {
-    if (isStarkRootDomain(identity?.domain ?? "")) {
-      const refreshData = () =>
-        fetch(
-          `${process.env.NEXT_PUBLIC_STARKNET_ID_API_LINK}/domain_to_data?domain=${identity.domain}`
-        )
-          .then(async (response) => {
-            if (!response.ok) {
-              throw new Error(await response.text());
-            }
-            return response.json();
-          })
-          .then((data: Identity) => {
-            setApiIdentity(data);
-          });
-      refreshData();
-      const timer = setInterval(() => refreshData(), 30e3);
-      return () => clearInterval(timer);
+    if (isStarkRootDomain(identity?.domain.domain)) {
```

Comment:
`.domain.domain` ? Are sure that works ?

Reply:
yes
the new format looks like this 

```
{
    "id": "0x000000000000000000000000000000000000000000000000000000bbf5ff52f6",
    "owner": "0x04620172a73caa49fdf729e258c0cd7b1f6e4262b8a7e724877af5f780806b40",
    "main": false,
    "creation_date": 1695908102,
    "domain": {
        "domain": "ayushtom.stark",
        "migrated": false,
        "root": true,
        "creation_date": 1695908102,
        "expiry": 1766756102,
        "resolver": null,
        "legacy_address": "0x04620172a73caa49fdf729e258c0cd7b1f6e4262b8a7e724877af5f780806b40",
        "rev_address": "0x04620172a73caa49fdf729e258c0cd7b1f6e4262b8a7e724877af5f780806b40"
    },
    "user_data": [],
    "verifier_data": [
        {
            "verifier": "0x06ac597f8116f886fa1c97a23fa4e08299975ecaf6b598873ca6792b9bbfb678",
            "field": "0x000000000000000000000000000000000000000000000000000000006e616d65",
            "data": "0x00b9093b37aa3450c05a7739ea0f0a51d730719f777ca1199e13e1d6f466add4"
        },
        {
            "verifier": "0x002d5f0f2fcba62a54f0d8a73fb2e437d76c53f7297153b2afd1f198af6d12aa",
            "field": "0x0000000000000000000000000000000000000000000000000074776974746572",
            "data": "0x00000000000000000000000000000000000000000000000011472ee288547006"
        },
        {
            "verifier": "0x002d5f0f2fcba62a54f0d8a73fb2e437d76c53f7297153b2afd1f198af6d12aa",
            "field": "0x0000000000000000000000000000000000000000000000000000676974687562",
            "data": "0x00000000000000000000000000000000000000000000000000000000027be78a"
        },
        {
            "verifier": "0x002d5f0f2fcba62a54f0d8a73fb2e437d76c53f7297153b2afd1f198af6d12aa",
            "field": "0x00000000000000000000000000000000000000000000000000646973636f7264",
            "data": "0x00000000000000000000000000000000000000000000000008929286c400000a"
        },
        {
            "verifier": "0x0293eb2ba9862f762bd3036586d5755a782bd22e6f5028320f1d0405fd47bff4",
            "field": "0x0000000000000000000000000070726f6f665f6f665f706572736f6e686f6f64",
            "data": "0x0000000000000000000000000000000000000000000000000000000000000001"
        }
    ],
    "extended_verifier_data": []
}

Reply:
Then we need to rename the first domain here imo. 

`domainInfos` for example cause otherwise the dev has to log it to understand